### PR TITLE
Fixed hindi spellings of countries

### PR DESCRIPTION
**Pull request by tanishqchamola**
Hi,
I have fixed the spellings of the country names in Hindi language. Please merge it.

### CODE REVIEW

File: app/config/locale/translations/hi.countries.php
Comment (spielers) [inline app/config/locale/translations/hi.countries.php:48]: गणराज्या = >गणराज्य

**Comment by punitdiwan** [comment]
@tanishqchamola The hindi spelling of Afganistan is अफ़ग़ानिस्तान, its not अफ़्गानिस्तान. Please add dot (nukta) in  ग. Please check in hindi version of Wikipedia about Afganistan.

File: app/config/locale/translations/hi.countries.php
Comment (spielers) [reply app/config/locale/translations/hi.countries.php:48]: Or we can use चेक गणतंत्र

**Review by spielers — CHANGES_REQUESTED** [review]

**Comment by spielers** [comment]
@tanishqchamola  Thanks for your contributions But I would suggest you follow [WIKIPEDIA ](https://hi.wikipedia.org/wiki/%E0%A4%B5%E0%A4%BF%E0%A4%B6%E0%A5%8D%E0%A4%B5_%E0%A4%95%E0%A5%87_%E0%A4%B8%E0%A4%AD%E0%A5%80_%E0%A4%A6%E0%A5%87%E0%A4%B6)guideline and translation

File: app/config/locale/translations/hi.countries.php
Comment (spielers) [inline app/config/locale/translations/hi.countries.php:190]: सन्त ==> सेण्ट

**Review by spielers — CHANGES_REQUESTED** [review]
Hey @tanishqchamola follow Wikipedia for more correction

**Comment by tanishqchamola** [comment]
@spielers Fixed the mistakes, please take a look. :)

**Comment by spielers** [comment]
Hey, @tanishqchamola  Thanks for your contribution. I would suggest you should have a look at our open issues #25 #546 
if you are aiming for hacktoberfest, according to new rules of hacktoberfest this PR is not considered as a valid PR 

Best,
spielers

**Comment by tanishqchamola** [comment]
@spielers, I am glad that I could contribute but the first point in issue #25 stated that we can submit PRs with fixes and improvements. So I thought it would be valid for hacktoberfest.

Screenshot 2020-10-08 at 10 58 09

**Comment by spielers** [comment]
Hey,
That is great point we have updated that list before hacktoberfeast newer
update. will update that list and issue ASAP

Thanks for your response

Best,
Spielers

**Comment by spielers** [comment]
hey, @tanishqchamola  You got you covered I had a discussion With the maintainer team, will get you merged asap

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Comment by tanishqchamola** [comment]
Thanks a lot @eldadfux. Will connect to you on discord. 👍

**Pull request merged**
